nemo_text_processing/text_normalization/hu/data/electronic/symbols.tsv

```diff
@@ -0,0 +1,23 @@
+.	pont
+-	kötőjel
+_	aláhúzásjel
+!	felkiáltójel
+#	kettőskereszt
+$	dollárjel
+%	százalékjel
+&	és
+'	aposztróf
+*	csillagjel
++	pluszjel
+/	perjel
+\	fordított perjel
+=	egyenlő
```

Comment:
It is not incorrect just not consistent with the other. I would use: egyenlőségjel